### PR TITLE
React Native Release 4.1.0

### PR DESCRIPTION
**Pull request by DmitriyKirakosyan**
* [x] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [ ] Are the files formatted correctly?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Bumped the versions
Updated TestApp to use latest SDKs
Updated the CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # App Center SDK for React Native Change Log
 
-## Version 4.0.3 (Under development)
+## Version 4.1.0
```
Comment (MatkovIvan) [inline CHANGELOG.md:3]: You must copy related changes from native SDKs

**Review by MatkovIvan — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (charmian01) [reply CHANGELOG.md:3]: Updated CHANGELOG for this, please help to check.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,15 @@
 
 * **[Feature]** Use XCFramework format for the AppCenterReactNativeShared pod, it allows running the SDK on Apple Silicon simulators.
 * **[Feature]** Support Mac Catalyst. Cocoapods 1.10.1+ is required.
+* **[Improvement]** Use ASWebAuthenticationSession for authentication on iOS 12 or later.
+* **[Fix]** Fix Objective-C properties attributes warnings in MRC projects.
```
Comment (MatkovIvan) [inline CHANGELOG.md:12]: Not related to RN

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,15 @@
...
+
+### App Center Crashes
+
+#### Android
+
+* **[Fix]** Fix formatting of stack trace in the `ErrorReport`.
+* **[Fix]** Fix setting `userId` value in NDK crashes before sending.
```
Comment (MatkovIvan) [inline CHANGELOG.md:19]: Not related to RN

File: CHANGELOG.md (hunk 2 above)
Comment (charmian01) [reply CHANGELOG.md]: Removed.

File: CHANGELOG.md (hunk 3 above)
Comment (charmian01) [reply CHANGELOG.md]: Removed.

File: CHANGELOG.md (hunk 1 above)
Comment (russelarms) [reply CHANGELOG.md:3]: Don't we need to add these changes about Android.Crashes too?
https://github.com/microsoft/appcenter-sdk-android/blob/develop/CHANGELOG.md#app-center-crashes
```
[Fix] Fix formatting of stack trace in the ErrorReport.
[Fix] Fix setting userId value in NDK crashes before sending.
```

**Review by russelarms — DISMISSED** [review]

**Review by russelarms — APPROVED** [review]

**Pull request merged**